app/src/i18n/es/index.json

```diff
@@ -79,6 +79,7 @@
   "deleted": "Eliminada",
   "payer": "Pagador/a",
   "payee": "Cobrador/a",
+  "feedback": "Comentarios",
```

Comment:
I think that "Comentarios" don't really work very well in spanish. There is no direct translation, but maybe I'd be more inclined to "Opina", which is "Give your opinion".